src/thermo/PlasmaPhase.cpp

```diff
@@ -677,4 +722,41 @@ void PlasmaPhase::getGibbs_RT(double* grt) const
     }
 }
 
+double PlasmaPhase::electronMobility() const
+{
+    // Only implemented when using the Boltzmann two-term EEDF
+    if (m_distributionType == "Boltzmann-two-term") {
+        if (!m_eedfSolver) {
+            throw CanteraError("PlasmaPhase::electronMobility",
+                "EEDF solver is not initialized.");
+        }
```

Comment:
I think this case is impossible, given that `m_eedfSolver` is initialized in the `PlasmaPhase` constructor.